packages/docs/src/routes/qwikcity/routing/index.mdx

```diff
@@ -50,8 +50,8 @@ src/
 Special named directories with square brackets, such as `[paramName]` and `[...catchAll]` can be used to match route segments which are dynamic:
 
 ```
-src/routes/blog/index.tsx → /blog/:slug (/blog/hello-world)
-src/routes/user/[username]/index.tsx → /user/:username/settings (/user/foo/)
+src/routes/blog/index.tsx → /blog
+src/routes/user/[username]/index.tsx → /user/:username (/user/foo)
 src/routes/post/[...all]/index.tsx → /post/* (/post/2020/id/title)
```

Comment:
I'd replace `/post/*` part with something more standard than it is now. Not sure how though. Maybe:

```
/post/:all (/post/2020/id/title)
```

But that does not indicate catch-all routes that can be deeper than one level. Or:

```
/post/:...all (/post/2020/id/title)
```

though this one ☝️ looks ugly. Or maybe:

```
/post/[:all] (/post/2020/id/title)
```

[Next.js](https://nextjs.org/docs/routing/dynamic-routes#optional-catch-all-routes) uses this:

```
/post/[[...all]] (/post/2020/id/title)
```

Though, I'd rather go with `:` instead of `...`:

```
/post/[[:all]] (/post/2020/id/title)
```

@zanettin any suggestion? 😄

Reply:
good question 🤔 i'd go with v1 (`:all`). in the code you still have a single string containing the whole path. guess the notations in brackets could be misleading. just my thoughts.

Reply:
agree 👍 let's keep the wildcard for now. should be clear in the full context 🙏